raiden_contracts/data_0.10.0/gas.json

```diff
@@ -0,0 +1,21 @@
+{
+    "EndpointRegistry.registerEndpoint": 48408,
+    "MonitoringService.claimReward": 42679,
+    "MonitoringService.monitor": 201313,
+    "OneToN.claim": 90897,
+    "SecretRegistry.registerSecret": 45757,
+    "TokenNetwork DEPLOYMENT": 4031160,
+    "TokenNetwork.closeChannel": 111226,
```

Comment:
Ah, my bad above. We have a file per version now.

Reply:
Ah, I replied before seeing this.

Reply:
One disadvantage of "everything per version" is a big release PR like this one.